### PR TITLE
Add new bot - iam_user_detach

### PR DESCRIPTION
**Pull request by TinaSobol**

### CODE REVIEW

File: bots/iam_user_detach.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+def run_action(boto_session, rule, entity, params):
+    text_output = ''
+    user_to_remove = entity.get('name')
+
+    # Get event from cloud trail
+    event = bots_utils.cloudtrail_event_lookup(boto_session, entity, EVENT_NAME,
+                                               resource_name_to_filter=user_to_remove)
+    if not event:
+        # If valid event was not found - bot will not run
+        text_output = f'Error: No matching \'{EVENT_NAME}\' events were found in cloud trail. Bot wasn\'t executed.'
```
Comment (RavidAtias) [inline bots/iam_user_detach.py:29]: No need to use those \, it's more elegant to use {}

File: bots/iam_user_detach.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    return text_output
+
+
+def remove_user_from_group(boto_session, user_to_remove, group_name):
+    iam_client = boto_session.client('iam')
+    try:
+        # Remove the IAM user from the IAM group
+        iam_client.remove_user_from_group(GroupName=group_name, UserName=user_to_remove)
+    except botocore.exceptions.ClientError as e:
+        return 'Client error: %s \n' % e
```
Comment (RavidAtias) [inline bots/iam_user_detach.py:45]: use the f'' syntax

**Review by RavidAtias — CHANGES_REQUESTED** [review]

File: bots/iam_user_detach.py (hunk 1 above)
Comment (TinaSobol) [reply bots/iam_user_detach.py:29]: Did you mean -  \'   ?

Comment (RavidAtias) [reply bots/iam_user_detach.py]: I meant - \

File: bots/iam_user_detach.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    try:
+        # Remove the IAM user from the IAM group
+        iam_client.remove_user_from_group(GroupName=group_name, UserName=user_to_remove)
+    except botocore.exceptions.ClientError as e:
+        return f'Client error: {e} \n'
+    else:
+        return f'Success: The user: {user_to_remove} was successfully detached from the group: {group_name}'
+
+
+def get_details_from_event(event):
```
Comment (doronzil) [inline bots/iam_user_detach.py:50]: what if the parsing fails?

**Review by doronzil — CHANGES_REQUESTED** [review]

**Review by RavidAtias — APPROVED** [review]

**Review by doronzil — APPROVED** [review]

**Pull request merged**